src/main/java/it/gov/pagopa/pu/debtpositions/repository/InstallmentNoPIIRepository.java

```diff
@@ -19,5 +19,14 @@ public interface InstallmentNoPIIRepository extends JpaRepository<InstallmentNoP
   @Query("UPDATE InstallmentNoPII i SET i.status = :status, i.iupdPagopa = :iupdPagopa WHERE i.installmentId = :installmentId")
   void updateStatusAndIupdPagopa(@Param("installmentId") Long installmentId, @Param("iupdPagopa") String iupdPagopa, @Param("status") InstallmentStatus status);
 
-
+  @Query("""
+    SELECT COUNT(dp)
```

Comment:
I don't remember if count accept some parameter, evenutally let's count the installments
```suggestion
    SELECT COUNT(i)
```